products/sce/autoscaling/autoscaling.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Autoscaling Overview'
+---
+
+The new **Autoscaling** feature in SaladCloud dynamically adjusts the number of container instances based on the number
+of messages in a queue. This ensures that your infrastructure scales up when demand increases and scales down during low
+usage periods, optimizing both performance and cost efficiency.
+
+To use the autoscaling feature, you need to integrate the Salad queue into your container groups. The system will
+monitor the queue length and adjust the number of container replicas accordingly.
+
+## Prerequisites
+
+Before you can enable autoscaling, you must configure the Salad Queue:
+
+1. Ensure that the [**Salad Queue**](/products/sce/job-queues/job-queues) is enabled for your container group.
+2. Verify that your application can handle scaling with multiple container instances.
+3. Review the [Autoscaling Settings](/products/sce/autoscaling/settings) to understand how to fine-tune autoscaling for
+   your workloads.
+
+## Example Use Case
+
+Let's say you run a batch job, but the number of jobs you receive is inconsistent. Sometimes, there are no jobs at all,
+and you want to scale down to zero instances to avoid paying for unused compute resources. By setting the **Minimum
+Replicas** to 0, the system will automatically scale down to zero when the queue is empty, which means you don't pay for
+any resources when there is no workload. However, this may result in longer startup times when new jobs arrive, as the
```

Comment:
Do we have a docs link that we can share that explains the cold start times in more detail? Feeling like just leaving a developer with "longer startup times" might deter usage, similar to the confusion Speechify had with limited context.